client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -523,7 +523,7 @@
       },
       jurisdiction: {
         get() {
-          return this.$store.getters.jurisdiction
+          return this.$store.getters.jurisdiction && this.$store.getters.jurisdiction.toUpperCase()
```

Comment:
This (probably) returns undefined if there is no jurisdiction. Is that OK?

Reply:
This is ok, just ensuring it is Uppercase when it exists works. 
I would have optionally chained however it isn't supported in the current version :P